backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/MapFromCsv.kt

```diff
@@ -44,7 +44,7 @@ class MapFromCsv(config: ImportConfig, private val logger: Logger) :
                 it.email.clean(),
                 it.telephone.clean(),
                 it.homepage.clean(),
-                it.discount.clean(false),
+                (it.discountDE + "\n\n" + it.discountEN).clean(false),
```

Comment:
here should be checked if `it.discountEN` is null before adding it

Reply:
sth like 
```
      val discount =
            if (it.discountEN == null) {
                it.discountDE
            } else {
                it.discountDE + "\n\n" + it.discountEN
            }
```

Reply:
Thanks, a bit unfortunate that Kotlin does not give a compile time error :/